### DISCUSSION TITLE
Installation of capytaine

### DISCUSSION THREAD

**Discussion by vaibhavraghavan** [capytaine/capytaine#162] [General]
Hi,

I tried installing capytaine 1.3 and 1.2.1 using pip, but it gives me the same error:

  
  CPU dispatch  :
    Requested   : 'max -xop -fma4'
    Enabled     : SSSE3 SSE41 POPCNT SSE42 AVX F16C FMA3 AVX2 AVX512F AVX512CD AVX512_SKX AVX512_CLX AVX512_CNL AVX512_ICL
    Generated   : none
  CCompilerOpt.cache_flush[809] : write cache to path -> C:\Users\VRAGHA~1\AppData\Local\Temp\pip-install-814tdfrs\capytaine_832a8920cb7e46d1972bd625c79d0e48\build\temp.win-amd64-cpython-37\Release\ccompiler_opt_cache_ext.py
  [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure

Encountered error while trying to install package.

capytaine

note: This is an issue with the package mentioned above, not pip.
hint: See above for output from the failure.

--------------------------------------------------------------------------------------------------------------

I also tried cloning the package through git, but then I see some import issues pertaining to the fortran file XieDelhommeau_f90 which doesn't exist in the package. How can I install capytaine?

Regards,

Vaibhav

**Comment by mancellin** [comment]
Most likely due to pip not finding a compiler to compile the Fortran modules (`Delhommeau_f90` and `XieDelhommeau_f90`) within Capytaine. Setting up the compiler can be tricky on Windows.
The easiest solution might be to install Capytaine using the conda package as suggested in the documentation. Unlike the pip packages, the conda package comes with a Fortran module already compiled for Windows.
